README.md

```diff
@@ -2,8 +2,7 @@
 
 [![Project Status: WIP – Initial development is in progress, but there has not yet been a stable, usable release suitable for the public.](https://www.repostatus.org/badges/latest/wip.svg)](https://www.repostatus.org/#wip)
 
-
-[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=613269999&machine=premiumLinux&devcontainer_path=.devcontainer%2Fdevcontainer.json&location=WestUs2)
+<a href='https://codespaces.new/r-devel/r-dev-env'><img src='https://github.com/codespaces/badge.svg' alt='Open in GitHub Codespaces' style='max-width: 100%;'></a>
```

Comment:
Any reason to use HTML markup here?

I think you are right that we shouldn't add options to the url (so these can be set regardless of how people open the codespace), but I think you can just change the url in the previous code?

```suggestion
[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/r-devel/r-dev-env)
```

Reply:
Also, it would be good if you could update the screenshot in this PR to match what you see when clicking this link. I.e. the screenshot on this line: https://github.com/r-devel/r-dev-env/blob/main/README.md?plain=1#L19